test/tests.bats

```diff
@@ -195,7 +195,15 @@
 # accounts
 #
 
+@test "checking accounts.cf: if present shouldn't start the container" {
+  cp `pwd`/postfix/disab.accounts.cf `pwd`/postfix/accounts.cf
+  run docker run --rm --name test_on_accounts -v "$(pwd)/postfix":/tmp/postfix `docker inspect --format '{{ .Config.Image }}' mail`
+  [ "$status" -eq 1 ]
+  [ "${lines[1]}" = "SECURITY WARNING ==> ABORTED startup !" ]
```

Comment:
I'll be more explicit in the exit message.
Something like

```
#
# ERROR:
# Container startup has been aborted because password file containing clear entries has been provided
# Please have a look to the documentation to provide a crypted mail account database.
#
```